ui/app/controllers/jobs/run.js

```diff
@@ -30,8 +33,9 @@ export default Controller.extend({
     }
 
     try {
-      const planOutput = yield this.get('model').plan();
-      this.set('planOutput', planOutput.Diff);
+      yield this.get('model').plan();
+      const plan = this.get('store').peekRecord('job-plan', this.get('model.id'));
```

Comment:
Did you consider returning the JobPlan record from the adapter `plan` method? Reading this code, I had to go through the model to the adapter to verify that the `peekRecord` would work at this point. Returning a promise with a value instead might make the responsibility of the `plan` method clearer.